messaging/nostrelay/nostrelay.go

```diff
@@ -66,45 +66,44 @@ var upgrader = websocket.Upgrader{
 	CheckOrigin:     func(r *http.Request) bool { return true },
 }
 
-func initRelays(r []string) nostr.RelayPool {
```

Comment:
HI, I noticed the change here while im trying to reproduce the problem #3. It seems that currently configure wont influence how many relays being connected but only connect to wss://nostr.688.org/, is this some trade off because of the original go-nostr package? @gazhayes can i change it back?

Reply:
@pqingshuang I might have broken something lol
checking now! sorry

Reply:
@pqingshuang I have "fixed" this in #8 
I think it should be more clear now, can you have a look?